ProjectLighthouse.Servers.Website/Pages/Login/BannedUserPage.cshtml

```diff
@@ -30,6 +32,12 @@
                 }
             </span>    
         }
+        else
+        {
+            <span>
+                @Model.Translate(ModerationStrings.SuspensionExpiration, Model.Translate(GeneralStrings.Unknown))
+            </span>
+        }
 
         <ul>
             <li>@Model.Translate(ModerationStrings.LbpOnlineMultiplayer)</li>
```

Comment:
GitHub doesn't let me comment on other lines but on line 53 there is no handler for if a reason isn't given, it just looks like `Reason for suspension: ""`